src/database/factories/user.factory.ts

```diff
@@ -1,18 +1,24 @@
 import * as Faker from 'faker';
 import { define } from 'typeorm-seeding';
 import { User } from '@entities/user.entity';
+import { genSaltSync, hashSync } from 'bcrypt';
+
+const genders = ['female', 'male'];
```

Comment:
I think this can be moved to `user.entity`. Also, I would check if this kink be a type in order to validate we just accept those values

Reply:
Right, I was checking the posibility to implement it using a type defined by me on the database and also a Typescript Type would fit better. I'll do the changes :)